File: thereseWebPage.Core/DomainModel/Blog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace thereseWebPage.Core.DomainModel
{
   public class Blog
   {
       public string HeadLine { get; set; }
       public string BlogContent { get; set; }
       public object Picture { get; set; }
       public DateTime PublishedDate { get; set; }

       public Blog()
       {
           
       }
       public Blog(string headLine, string blogContent, object picture, DateTime publishedDate)
       {
           HeadLine = headLine;
           BlogContent = blogContent;
           Picture = picture;
           PublishedDate = publishedDate;
       }
   }
}

File: thereseWebPage.UserInterface/wwwroot/views/blog.js
function blogPage() {

    let blogComitments = model.pagesContent.blogPage;

    const blogs = blogComitments.map(com => `
            <div><hr>    ${com.headLine} <br>
                  Innhold: ${com.blogContent} <br>
                Bilde: ${com.picture}<br> 
                Dato: ${com.publishedDate} 
        </div>`).join('');
    let html = ` <h1>Blog Innlegg</h1>
            
    ${blogs}
    `;
    console.log(blogComitments, 'viser denne noe?');


    document.getElementById('mainContent').innerHTML = html;


}